README.md

```diff
@@ -1 +1,39 @@
-# publish-regulatory-artifacts
\ No newline at end of file
+# Publish regulatory artifacts
+
+Publishes the regulatory artifacts to the correct location to be read by the Software Lifecycle Tool.
+
+- Uploads test results to the targeted regulatory S3 bucket
```

Comment:
(Minor) I'd remove the `-` for this. No need to show this a listing. just add a `:` after the bucket.

Reply:
Ahh yes thank you for the tip.
It started out as a list since I wanted to implement the publishing of another artifact. But that is for another time 😅
https://qompium.atlassian.net/browse/RSR-32
So the list will be completed at a later moment 😄